### ISSUE TITLE
how to re-initialise/clean the database?

### ISSUE THREAD

**Issue by kbuggenhout** [intel-hpdd/intel-manager-for-lustre#334]
### Description 

when testing and intentionally breaking things like destroying a zpool with MGS still defined.

### Repro

What steps were taken leading to this issue
remove filesystem, zpool import mgt, zpool import mdt, zpool destroy mdt, zpool destroy mgt, ...
iml still has knowledge of mgt, while device no longer available. tried to remove it. chroma -f target remove [id] , waits forever... with -n goes in background.. and never stops trying.
 
### Version

What version of IML? 4.0.0.0 rc1
What Operating System? centos 
What Operating System version? 7.3

**Comment by kbuggenhout** [comment]
fixed by altering the database, updating the table chroma_core_managedtargetmount : 
    update chroma_core_managedtarget set not_deleted = 'f' where not_deleted = 't';
there was only one element with not_deleted set to false.
I also update the value, state in the same table to removed.

**Comment by kbuggenhout** [comment]
tested by creating a new FS on the config.

:)

**Comment by jgrund** [comment]
Interesting, I haven't seen this locally; though I am testing with a newer build. 



Was this a managed mode install? 
Did the mdt pool disappear and you just saw partitions?

I'd be interested if you still still see this with our upcoming build.

**Comment by kbuggenhout** [comment]
managed mode install with ZFS, but on virtual box, I suspect issues in the virtualbox config as it refused to see certain volumes as HA ( although they were configured to be shared) I chose remove filesystem, but for some reason the MGS remained. with its pacemaker config.
I could not remove the MGS config from IML gui.

I did manually stop mgs resource in pcs first, ( pcs resource disable [MGS_####], removed the resource) then imported the pool again and destroyed it. IML still had it as mounted and active.. while the resource no longer existed in pcs. 

I most likely took a short cut too many, dont think this should be considered as a bug, rather a side effect of the testing environment.

I am going to test that on our testbed in the HPC lab in Austin ( I work for DellEMC)

this is the actual installed version:
chroma-manager-4.0.0.0-4951.46.g0f1b2a7.jenkins_manager_for_lustre_1686.el7.x86_64
chroma-manager-libs-4.0.0.0-4951.46.g0f1b2a7.jenkins_manager_for_lustre_1686.el7.x86_64
chroma-manager-cli-4.0.0.0-4951.46.g0f1b2a7.jenkins_manager_for_lustre_1686.el7.x86_64
